internal-enrichment/virustotal/src/virustotal/builder.py

```diff
@@ -72,13 +71,20 @@ def _compute_score(stats: dict) -> int:
         int
             Score, in percent, rounded.
         """
-        return round(
+        vt_score = round(
             (
                 stats["malicious"]
                 / (stats["harmless"] + stats["undetected"] + stats["malicious"])
             )
             * 100
         )
+        if self.replace_with_lower_score:
+            return vt_score
+        self.create_note(
```

Comment:
I guess the note is to keep track of the VirusTotal score if we don't use it to update the score of the observable. However, if `replace_with_lower_score=false` and `vt_score > current score`, we will update with the VirusTotal score and have the note with the same score. Is it the expected behaviour?

Reply:
Good point -- fixing.